docs/part1.md

```diff
@@ -33,18 +33,490 @@ The landing NOMAD Entries page is a very intuitive and easy-to-use Graphical Use
+    Conversely, if the statistics are ever the browser down, deactivate them by deselecting "Filters" > "options menu" (`⋮`) > "Show advanced statistics".
+
+Which subsequent filter to search for first depends on your priorities: do you want to learn mostly about running Molecular Dynamics or how a forcefield for water should be tweaked?
+Let us go with former.
+Fortunately, there already is a **filter subgroup called "Molecular Dynamics", so click it open**.
+You are now presented with several settings.
+Judging from the statistics, we have the best chance with a setting that covers the widest range of calculations (28.000 entries).
+**Select "Temperature" under "Available Properties"**. <!-- @Joseph: do you want me to to add a "Molecular Dynamics" selection button?-->
+
+!!! tip
+    If you have trouble **understanding any term**, just hover over it with the mouse.
+    A summary text will appear.
+    For example, "Molecular Dynamics" > "Available Properties" clarifies that we are dealing observables documented along the trajectory.
+
+System specifications are found under "Material".
+Since we know the composition of our system, **click on "Elements / Formula"** and **select the matching elements** in the periodic table.
+The number of database entries containing the other elements too is communicated via the color code.
+Those that are already **filtered out will be gray**, just as you saw when opening the side pane.
+This is the filter stacking mentioned before, showing that Gromacs users typically work on organic systems.
+You can see both chips on the left.
+Note how there is an "and" in between.
+While **filters** between side menu options stack, those **within a side pane each have their own logic**.
+We are looking for pure water, so O and H suffice. <!-- Shame how we don't have a subgroup search yet -->
+**Remove all data with additional elements** by selecting "only compositions that exclusively contain these atoms".
+
+!!! success
+    You are left with 2 MD calculations in the entries list.
+    Clicking on either, will fold out a quick summary.
+    **Clicking on the arrow** (`->`) will bring you to an _Overview page_.
+    More on that in [Part II](part2.md).
+    By **clicking through on "FILES"**, you will get an overview of the uploaded files.
+    These for sure contain output, but often also input.
+    Both could serve as great starting points for deploying your own calculations.
+
+Actually, the **exact historical order** in which we applied the filters **does not matter**.
+I.e. the same set of filters will yield the same search result.
+This insight comes with a small caveat: the statistics that we see as we are stacking up our filters will behave differently.
+These filters are like small questions or details that we use construct an overall query.
+If we take a **different route**, we will see **different intermediate results** and this could lead us to **change our final formulation**.
```

Comment:
Modified the end to "If we take a different route, we will see different intermediate results and this could lead us to decide on different filters."
I also shuffled the phrases in this paragraph for clarity